src/main/kotlin/app/revanced/patches/youtube/layout/returnyoutubedislike/fingerprints/ShortsTextComponentParentFingerprint.kt

```diff
@@ -0,0 +1,19 @@
+package app.revanced.patches.youtube.layout.returnyoutubedislike.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+object ShortsTextComponentParentFingerprint : MethodFingerprint(
```

Comment:
This fingerprint is resolved from all classes and methods on YouTube. It looks quite weak due to its size, was this considered?